src/ape_test/plugin.py

```diff
@@ -38,11 +39,17 @@ def is_module(v):
     config.option.verbose = config.getoption("capture") == "no"
 
     # Inject the runner plugin (must happen before fixtures registration)
-    session = PytestApeRunner(config, project, networks, chain)
+    ManagerAccessBase.config_manager = config
```

Comment:
Are the injections here still needed? I would think this would already be done at this point?